migrations/20200730_01_create_table_users.py

```diff
@@ -0,0 +1,12 @@
+from yoyo import step
+
+__depends__ = {}
+
+steps = [
+    step("CREATE TABLE users (id integer PRIMARY KEY,"
```

Comment:
Can you please remind me where did we have a documentation about this structure?